Library/Formula/postgresql.rb

```diff
@@ -96,18 +86,17 @@ def install
     system "./configure", *args
     system "make", "install-world"
 
-    if build.with? "pgroonga"
-      resource("pgroonga").stage do
-        ENV.append_path "PATH", bin
-        system "make"
-        system "make", "install"
-      end
-    end
+    # move all of binaries to libexec/bin
+    (libexec/"bin").install Dir["#{bin}/*"]
   end
 
   def post_install
+    # hardlink binaries in libexec/bin to HOMEBREW_PREFIX
+    # https://github.com/Homebrew/homebrew/issues/45607
+    ln Dir["#{libexec}/bin/*"], HOMEBREW_PREFIX/"bin", :force => true
```

Comment:
Does this need to be done for all binaries or is there a subset we can use for e.g. knowing the extensions path?